lib/wallet/wallet_dashboard.dart

```diff
@@ -65,7 +64,7 @@ class _WalletDashboardState extends State<WalletDashboard> {
           icon: const Icon(Icons.link))));
     }
 
-    if (bitcoinBalance.total().asSats != 0 && lightningBalance.amount.asSats == 0) {
+    if ((bitcoinBalance.total().asSats > 0 && !channel.isAvailable()) || channel.isInitialising()) {
```

Comment:
was your intention to have the "Open Channel' button available when the channel is already initialising?

In case it wasn't, then I've reworked it slightly to disable button when it's initialising too:

```suggestion
    if (bitcoinBalance.total().asSats > 0 && !(channel.isAvailable() || channel.isInitialising())) {
```

Reply:
Good catch!

Reply:
@bonomat  @klochowicz 

well it was intentional! I wanted to show the card when it is initialising, giving the user feedback that the channel is still in establishment. The card is disabled if the channel is initialising